pkg/server/token_minter.go

```diff
@@ -292,6 +299,48 @@ func (s *TokenMintServer) processRequest(r *http.Request, auditEvent *auditEvent
 	return http.StatusOK, accessToken, nil
 }
 
+// validateRepositories checks the set of requested repositories against the allow list
+// to verity that it is authorizaed. Response contains the list of allowed repositories
+// after wild card match expansion.
+func validateRepositories(allowed, requested []string) ([]string, error) {
+	repositories := []string{}
+	// Loop through all of the requested repositories to verifiy that are in the configured
+	// allow list
+	for _, repo := range requested {
+		matched := false
+		for _, allow := range allowed {
```

Comment:
Should we let allow be *?

Reply:
Personally I feel we need to be explicit here so that new repositories aren't automatically included in the allow list.

Reply:
sounds good.